src/components/atoms/Grid/Grid.tsx

```diff
@@ -34,7 +34,7 @@ export const Container: FC<GridProps> = forwardRef(
             width: '100%',
             maxWidth: `calc(1200px + ${value}px)`,
             py: `${value}px`,
-            px: [2, null, 4],
+            px: [3, null, 4],
```

Comment:
halo halo znowu zmiany paddingów? to bedzie miało wpływ na kandydata chyba :P

Reply:
no miałam zrobić 16px czyli tak, jak wcześniej było ;> @exihuatl

Reply:
Rozumiem, że 16px jest narysowane przez Radka, więc wszystko ok.